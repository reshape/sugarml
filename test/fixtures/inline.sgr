li: a(href='wow') content
li: .foo hello
li: a.foo hello
.foo: p.bar
