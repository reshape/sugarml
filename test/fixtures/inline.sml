li: a(href='wow') content
li: .foo hello
