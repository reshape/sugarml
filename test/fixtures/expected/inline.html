<li><a href="wow">content</a></li><li><div class="foo">hello</div></li>
